README.rst

```diff
@@ -199,7 +199,13 @@ cross platform performance.
 The ``/gpu/cuda/*`` backends provide GPU performance strictly using CUDA.
 
 The ``/gpu/magma`` backend relies upon the `MAGMA <https://bitbucket.org/icl/magma>`_ package.
-
+To enable the MAGMA backend, the environment variable ``MAGMA_DIR`` must point to the top-level
+MAGMA directory, with the MAGMA library located in ``$(MAGMA_DIR)/lib/``.
+By default, ``MAGMA_DIR`` is set to ``../magma``; to build the MAGMA backend
+with a MAGMA installation located elsewhere, create a link to ``magma/`` in libCEED's parent
+directory, or set ``MAGMA_DIR`` to the proper location.  Currently, both basis function order
+and the number of quadrature points in one dimension must be <= 10 for tensor bases in MAGMA.
+(Non-tensor bases do not have these limits.)
```

Comment:
Thanks @nbeams for your addition! 

I only have a comment. Up to this point we haven't really given details about bases and quadrature points. We start that FEM description in Chapter 3, Interface Concepts. I understand why you wanted to make sure this information is clear upfront. But perhaps there is a better place to present this bit of information?

Reply:
@valeriabarra I agree, but I was having trouble finding anywhere else to put it, either.  It would make sense in a section about the use of different backends (a "tips for use" section or something) -- maybe with a table or list of which backends have certain functions (since not all backends implement everything).  But that would be a pretty big undertaking to create and maintain as backend development continues...
I'm open to any and all suggestions!  Or we could just remove it altogether, I suppose.  We have a "launch failure" error message for that particular failure mode, though that's not the only way to end up with a launch failure.

Reply:
Yes, in that case, it is probably better to remove this detailed sentence in this introduction. Unless others have some other strong opinions. Thanks Natalie

Reply:
Removed in 23e70c689fd857fa761a9eeb6b51a14f56d78ea0